### PR TITLE
VC-2242 improve export template zip file name

### PR DESCRIPTION
**Pull request by OlegApanovich**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -63,7 +63,7 @@ hubtools
 
 cache/*
 !cache/.gitkeep
-devElements/*
+devElements
```
Comment (pavelthq) [inline .gitignore:66]: make separate PR for this

File: plugin-wordpress.php [hunk 2]
```diff
@@ -70,10 +70,15 @@
...
 /**
- * Plugin core prefix for options/meta and etc.
+ * Plugin core prefix for options/meta etc.
  */
 define('VCV_PREFIX', 'vcv-');
 
+/**
+ * Plugin brand prefix for user visible files, zip archives names etc.
+ */
+define('VC_PREFIX', 'vc-');
```
Comment (pavelthq) [inline plugin-wordpress.php:80]: no need to introduce new constant. use string value only inside related parts.

**Review by pavelthq — CHANGES_REQUESTED** [review]

**Comment by OlegApanovich** [comment]
Changes of PR moved to addon repository
